### DISCUSSION TITLE
dpgen init_bulk method stuck in 02.md

### DISCUSSION THREAD

**Discussion by wxm0703** [deepmodeling/dpgen#1037] [Q&A]
Hello, I'm a beginner of dpgen. I want to generate the init dataset of CH4(provided by your website example) by using init_bulk method, but it always stuck in 02.md. 
In "02.md", "deepmd" folder doesn't exist. I check the remote root and it seems like running AIMD sucessfully. But the OUTCAR cannot be passed to the local root.            
how can I solve this problem?

**Comment by HuangJiameng** [comment]
Is there any error message?

**Reply by wxm0703** [reply]
/var/spool/torque/mom_priv/jobs/2552.n1.SC: line 16: module: command not found
/var/spool/torque/mom_priv/jobs/2552.n1.SC: line 17: module: command not found

**Comment by HuangJiameng** [comment]
Could you provide the `.sub` script in the task path and your `json` files to help locate the problem?

**Reply by wxm0703** [reply]
Thanks for your reply, I have been resolved (change the batch_type from ”PBS“ to “Torque” in  machine.json) .
reference: https://github.com/deepmodeling/dpgen/discussions/914
